### ISSUE TITLE
Use external prop-types package 

### ISSUE THREAD

**Issue by dmcclory** [newbreedofgeek/react-stepzilla#37]
Hi!

I'm using react-stepzilla, but I'm encountering these warnings:

```
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
```

They're pretty trivial, but they're messing up my CI runs.

I checked the source and noticed that stepzilla is importing PropTypes from react, which is in the process of being moved into that external package.

I can put up a PR if you'd like!

**Comment by newbreedofgeek** [comment]
@dmcclory the latest version of React moved PropTypes to its own package:
https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html

I'll probably bump up the packages React version today and try and push a new version of StepZilla.

**Comment by newbreedofgeek** [comment]
@dmcclory I've upgraded to 15.5.4 and pulled out PropTypes to use prop-types npm.

This should dig rid of the warning messages for you now. Please let me know if you have any other issues.

New build has been published to npm, so make sure you update to v4.5.0

**Comment by newbreedofgeek** [comment]
@dmcclory closing as I've not heard anything and assume it worked.
